static/js/front.js

```diff
@@ -41,6 +42,10 @@ function contactForm () {
   })
 }
 
+function contactFormPost(){

```

Comment:
this function is useless

Reply:
That it is. Thanks for the observation.

I have submitted an updated pull request without it.

Reply:
Thanks for the tip. I got rid of it.